src/pages/parktaehwan/Main/Feeds/Comment.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import NewComment from './NewComment';
+
+class Comment extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      inputValue: '',
+      comments: [],
+      disabled: true,
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/commentData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(res => {
+        this.setState({
+          comments: res.first,
+        });
+      });
+  }
+
+  handleInputValue = e => {
+    this.setState({
+      inputValue: e.target.value,
+      disabled: false,
+    });
+  };
+
+  handleCommentValue = e => {
+    const commentNew = {
+      id: this.state.comments.length + 1,
+      cmtName: 'mekemeke',
+      cmtContent: this.state.inputValue,
+      like: false,
+    };
+    // this.state.comments.push(this.state.inputValue);
+    this.setState({
+      comments: [...this.state.comments, commentNew],
+      inputValue: '',
+      disabled: true,
+    });
+  };
+
+  removeComment = id => {
+    const filteredComments = this.state.comments.filter(
+      NewComment => NewComment.id !== id
+    );
+    this.setState({
+      comments: filteredComments,
+    });
+  };
+
+  clickUpload = e => {
+    this.handleCommentValue();
+  };
+
+  pressEnter = e => {
+    if (e.key === 'Enter') {
+      this.handleCommentValue();
+    }
+  };
+
+  likeComment = id => {
+    const newLikedComments = this.state.comments.map(el => {
+      if (el.id === id) {
+        el.like = !el.like;
+      }
+      return el;
+    });
```

Comment:
- el.like = !el.like 는 기존 state 에 직접 접근해서 값을 수정합니다. 불변성을 온전히 지키려면 다음과 같이 작성을 해주셔야 합니다.
```suggestion
    const newLikedComments = this.state.comments.map(el => {
      if (el.id === id) {
        return {...el, like: !el.like};
      }
      return el;
    });
```
- 좀 더 축약 하자면 다음과 같습니다!
```suggestion
    const newLikedComments = this.state.comments.map(el => {
      return el.id === id ? {...el, like: !el.like} : el;
    });
```